.github/workflows/commit-trailers.yml

```
@@ -0,0 +1,50 @@
+  pull_request:
+  merge_group:
+
+permissions:
+  contents: read
+  pull-requests: read
+
+jobs:
+  no-ai-coauthor:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Reject "Co-Authored-By: <AI>" trailers
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)